### PR TITLE
Query params

### LINKED ISSUE #62: Show query parameters as a list in the request detail view
They query parameters should be displayed as a list in the request detail view.

### PR DESCRIPTION
**Pull request by mitchhentges**
Fixes #62 
screenshot from 2016-02-01 10-33-36

### CODE REVIEW

File: src/component/inspect/key-value.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const {string} = React.PropTypes;
+
+export const KeyValue = ({_key, value}) => {
```
Comment (mitchhentges) [inline src/component/inspect/key-value.js]: Not super happy with the naming here, I'm open to suggestions

**Comment by nerdbeere** [comment]
This can get really long now. I think we need to implement a scrollbar or something like this

**Comment by mitchhentges** [comment]
Should that be part of this issue?

**Comment by nerdbeere** [comment]
In my opinion it should have at least a very simple solution.
I want to avoid breaking the master

**Comment by mitchhentges** [comment]
`overflow-y: scroll` set!

**Pull request merged**